static/src/javascripts/projects/common/modules/discussion/discussion-frontend.js

```diff
@@ -1,33 +1,39 @@
 define([
-    'common/utils/fetch-json',
     'common/utils/mediator',
     'common/utils/report-error'
 ], function(
-    fetchJson,
     mediator,
     reportError
 ) {
-    function canRun(ab, curlConfig) {
-        return (ab.isInVariant('DiscussionExternalFrontendAvatar', 'react') && curlConfig.paths['discussion-frontend-react']) ||
-            (ab.isInVariant('DiscussionExternalFrontendAvatar', 'preact') && curlConfig.paths['discussion-frontend-preact']);
-    }
+    var VARIANT = 'preact';
```

Comment:
just in case we want to use `'react'` in the future or you want to override it locally